xl-op/digitalai/dai-deploy/kubernetes-generic/dai-deploy_cr_default.yaml.tmpl

```diff
@@ -15,6 +15,7 @@ metadata:
   {{- end }}
   {{- end }}
 spec:
+  {{- if eq .IsCustomImageRegistry true }}
   global:
     {{- if eq .IsCustomImageRegistry true }}
```

Comment:
Its enough to enclose the entire global section alone with if "IsCustomImageRegistry"
We can remove the if condition here. And also the accompanying {{end}}.

Reply:
Same for release cr